website/docs/r/storage_account_local_user.html.markdown

```diff
@@ -0,0 +1,146 @@
+---
+subcategory: "Storage"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_account_local_user"
+description: |-
+  Manages a Storage Account Local User.
+---
+
+# azurerm_storage_account_local_user
+
+Manages a Storage Account Local User.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "WestEurope"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "example-account"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_kind             = "StorageV2"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  is_hns_enabled           = true
+}
+
+resource "azurerm_storage_container" "example" {
+  name                 = "example-container"
+  storage_account_name = azurerm_storage_account.example.name
+}
+
+resource "azurerm_storage_account_local_user" "example" {
+  name                 = "user1"
+  storage_account_id   = azurerm_storage_account.example.id
+  ssh_key_enabled      = true
+  ssh_password_enabled = true
+  home_directory       = "example_path"
+  ssh_authorized_key {
+    description = "key1"
+    key         = local.second_public_key
```

Comment:
```suggestion
    key         = local.first_public_key
```

Should this be `first_public_key`?

Reply:
This doesn't make a difference?

Reply:
Sure - but it's a bit confusing when reading the docs to see `second_public_key` and not `first_public_key` anywhere, it makes you think you've missed something.

What's the reasonable real world case where you'd add the same authorized key twice to the same storage account?

Reply:
Sorry, you are right! That is a mistake in my test. I intend to test multiple ssh key in this case: https://github.com/hashicorp/terraform-provider-azurerm/blob/779cf9d9be320c0a10cdd5715ffc7f89ff25f66e/internal/services/storage/storage_account_local_user_resource_test.go#L214-L234

Reply:
Cool; updating the docs examples to use both (or just one) would help reduce confusion too :)